cvat/apps/engine/media_extractors.py

```diff
@@ -662,7 +662,12 @@ def save_as_chunk(self, images, chunk_path):
                 output = io.BytesIO()
                 if self._dimension == DimensionType.DIM_2D:
                     pil_image = rotate_within_exif(Image.open(image))
```

Comment:
A separate question is where the rotation should be handled. But leave it as is for now.